impose/gui/colocalize.py

```diff
@@ -107,6 +108,12 @@ def add_paths(self, paths):
         for pp in paths:
             try:
                 self.session_scheme.append(pp)
+            except BmlabInvalidFileError as e:
```

Comment:
Same thing as in collect.py. There should be a way to deal with this in the format loader only.